bench/playbooks/roles/wkhtmltopdf/tasks/main.yml

```
@@ -20,11 +20,17 @@
     force: yes
   when: ansible_os_family == 'Debian'
 
-- name: download wkthmltox Ubuntu 20
+- name: download wkthmltox Ubuntu 20 amd64
   get_url:
     url: https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/0.12.5/wkhtmltox_0.12.5-1.focal_amd64.deb
     dest: /tmp/wkhtmltox.deb
-  when: ansible_distribution == 'Ubuntu' and ansible_distribution_major_version == '20'
+  when: ansible_distribution == 'Ubuntu' and ansible_distribution_major_version == '20' and ansible_architecture == 'x86_64'
+- name: download wkthmltox Ubuntu 20 arm64
```

```suggestion

- name: download wkthmltox Ubuntu 20 arm64
```

For consistent formatting